### PR TITLE
Add tests for source and loadedSources

### PR DESCRIPTION
**Pull request by jtpio**
This change adds two tests for the `DebugSession`.

This is in preparation for #169, and to test whether it is possible to retrieve the content for a `DebugProtocol.Source`.

At the moment the tests fail for both the `loadedSources` and `sources` request:

```
D00000.415: (while handling {'command': 'loadedSources', 'seq': 5, 'type': 'request'})
            IDE <-- {
                "type": "response",
                "seq": 9,
                "request_seq": 5,
                "success": false,
                "command": "loadedSources",
                "message": "Unknown command",
                "body": {}
            }
```

```
D00000.599: (requested while handling {'command': 'source', 'seq': 7, 'type': 'request'})
            PYD --> {
                "type": "response",
                "request_seq": 1000000005,
                "success": false,
                "command": "source",
                "body": {
                    "content": ""
                },
                "seq": 22,
                "message": "Source unavailable",
                "pydevd_cmd_id": 502
            }
```

### CODE REVIEW

**Comment by jtpio** [comment]
Will check what happens in VS Code and whether or not the `source` request is sent.

**Comment by jtpio** [comment]
Testing with:

`test.py`

```python
from mymodule import add

i = 0
i += 1
i += 2
i

add(i, 2)
```

`mymodule.py`

```python
def add(a, b):
    print(a)
    print(b)
    return a + b
```

Stepping into the `add` function triggers the `module` event, with the path to `mymodule.py`. But there is no `source` request, most likely because the file is local and VS code is able to open it.

**Comment by jtpio** [comment]
Rebased so we can use the latest `xeus-python` with support for `source`.

File: tests/src/session.spec.ts [hunk 1]
```diff
@@ -262,4 +262,28 @@ describe('protocol', () => {
...
     });
   });
+
+  describe('#loadedSources', () => {
+    it('should *not* retrieve the list of loaded sources', async () => {
+      // `loadedSources` is not supported at the moment "unknown command"
+      const reply = await debugSession.sendRequest('loadedSources', {});
+      expect(reply.success).to.be.false;
+    });
+  });
```
Comment (jtpio) [inline tests/src/session.spec.ts]: Leaving it in the tests so we know `loadedSources` is not supported.

Also fine to remove it if needed.

Comment (JohanMabille) [reply tests/src/session.spec.ts]: I think it's a good thing to keep it.

**Pull request merged**